src/wdio/services/ServeStaticService.js

```diff
@@ -2,19 +2,74 @@ import serveStatic from '../../../scripts/serve/serve-static';
 import SERVICE_DEFAULTS from '../../../config/wdio/services.default-config';
 import Logger from '../../../scripts/utils/logger';
 
+const WebpackDevServer = require('webpack-dev-server');
+const webpack = require('webpack');
+const ip = require('ip');
+
 const { serveStatic: SERVE_STATIC_DEFAULTS } = SERVICE_DEFAULTS;
 const context = '[Terra-Toolkit:serve-static-service]';
 
+const displayServer = (localAddress, networkAddress) => {
+  Logger.log('Server started listening at', { context });
+  Logger.log(`* Local:            ${Logger.emphasis(localAddress)}`);
+  Logger.log(`* On your network:  ${Logger.emphasis(networkAddress)}`);
+};
+
+// Create a webpack dev server instance.
+const wds = (options) => {
+  const {
+    port, index, locale,
+  } = options;
+  const host = '0.0.0.0';
+  let { config } = options;
+  // if config is a function, execute it with prod mode if applicable.
+  if (typeof config === 'function') {
+    const env = { ...locale && { defaultLocale: locale } };
+    config = config(env, { p: true });
+  }
+
+  // pull the dev server options out of the webpack config. override host, port, and stats. SRY.
+  const devServerOptions = Object.assign({}, config.devServer, {
+    // Disable hot reloading
+    hot: false,
+    inline: false,
+    host,
+    port,
+    index,
+    stats: {
+      colors: true,
+      children: false,
+    },
+  });
+
+  // get a compiler
+  const compiler = webpack(config);
+  // get a server
+  const devServer = new WebpackDevServer(compiler, devServerOptions);
+
+  // start that server.
+  return new Promise((resolve, reject) => {
+    devServer.listen(port, host, (err) => {
+      if (err) {
+        reject(err);
+      }
+      const localAddress = `http://${host}:${port}/`;
+      const networkAddress = `http://${ip.address()}:${port}/`;
```

Comment:
We should drop one of these lines. This service should be run in a docker container and the ip.address() value might actually be an value passed by a CI pipeline via [`process.env.WDIO_EXTERNAL_HOST`](https://github.com/cerner/terra-toolkit/blob/ce393c4ec9647cd83d0193d9f66add3c9995346e/config/wdio/wdio.conf.js#L10)

If we feel we should keep this log message, we should be able to pull the baseUrl form the wdio config since it will defined as [`baseUrl: `http://${ip}:${externalPort}`,`](https://github.com/cerner/terra-toolkit/blob/ce393c4ec9647cd83d0193d9f66add3c9995346e/config/wdio/wdio.conf.js#L10).

We have access to this from the onPrepare() func

Reply:
I'm happy dropping both, I kept them for passivity.